src/electron/views/device-connect-view/components/android-setup/device-description.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { NamedFC } from 'common/react/named-fc';
+import { DeviceMetadata } from 'electron/flux/types/device-metadata';
+import { Icon } from 'office-ui-fabric-react';
+import * as React from 'react';
+import * as styles from './device-description.scss';
+
+export interface DeviceDescriptionProps extends DeviceMetadata {}
+
+export const DeviceDescription = NamedFC<DeviceDescriptionProps>('DeviceDescription', props => {
+    const iconName: string = props.isEmulator ? 'Devices3' : 'CellPhone';
+    const iconAriaLabel: string = props.isEmulator ? 'Emulator' : 'Device';
+
+    return (
+        <React.Fragment>
```

Comment:
You can omit the `<React.Fragment>` wrapper here; it's the longhand way to write `<>` in a tsx file, but from a render function, it's only necessary if you need to have render return multiple root-level values

Reply:
Removed